node-tests/unit/engine-addon-test.js

```diff
@@ -6,12 +6,15 @@ const expect = require('chai').expect;
 describe('engine-addon', function() {
   describe('updateFastBootManifest', function() {
     it('adds necessary vendorFiles to the manifest when lazyLoading is enabled', function() {
+
+      /*eslint-disable*/
```

Comment:
Can we specify which rules are being disabled? Making it explicit will make it more understandable for developers that come along later.

Reply:
I tried just disabling the rule - but maybe I'm not doing it right  - this is the rule https://github.com/ember-cli/eslint-plugin-ember/blob/master/docs/rules/avoid-leaking-state-in-ember-objects.md

I tried `/* eslint-disable avoid-leaking-state-in-ember-objects */` but the rule was still getting triggered

Reply:
I figured it out - will update in a bit

Reply:
@trentmwillis is not it better to use `lazyLoading: true`? than removing the eslint